examples/src/ThumbnailCardExamples.js

```diff
@@ -43,6 +45,11 @@ const ThumbnailCardExamples = () => {
                 <br />
                 <ThumbnailCard highlightOnHover thumbnail={thumbnail} title={title} />
             </div>
+            <div>
+                <h1>Action Item Example</h1>
+                <br />
```

Comment:
Is this line break necessary?

Reply:
That was put there to provide a bit of spacing but I am cool with removing it here and in the other places above.

Reply:
Since the effect is purely visual, you could replace the line break with an inline style on the `h1` element.